siriuspy/siriuspy/orbintlk/main.py

```diff
@@ -356,7 +358,7 @@ def set_intlk_lims(self, intlk_lim, value):
 
     # --- reset ---
 
-    def cmd_reset(self, state):
+    def cmd_reset(self, state, value=None):
```

Comment:
if you are not going to use it (`value`), throw it away in the first line of the method.

Reply:
ok, done!